bindings/c/CMakeLists.txt

```diff
@@ -373,7 +373,7 @@ if(NOT WIN32)
         COMMAND ${CMAKE_SOURCE_DIR}/tests/TestRunner/upgrade_test.py
         --build-dir ${CMAKE_BINARY_DIR}
         --test-file ${CMAKE_SOURCE_DIR}/bindings/c/test/apitester/tests/upgrade/MixedApiWorkloadMultiThr.toml
-        --upgrade-path "7.1.9" "wiggle" "7.2.0"
+        --upgrade-path "7.1.9" "wiggle" "7.3.0"
```

Comment:
Are there any upgrade tests that go from 7.2 -> 7.3?

Reply:
We cannot have them yet. At first we need 7.2 to be released and its binaries made available for download